.github/workflows/deploy-vault.yml

```diff
@@ -0,0 +1,62 @@
+name: deploy-vault.ci
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
This should happen before the PR is merged.

Doing it on main will create 2 versions, the one with zeroAddress and then another version with address.